packages/application-shell/src/components/project-expired/project-expired.js

```diff
@@ -6,14 +6,19 @@ import ServicePageProjectSwitcher from '../service-page-project-switcher';
 import messages from './messages';
 
 const salesEmail = 'sales@commercetools.com';
-const mailto = <a href={`mailto:${salesEmail}`}>{salesEmail}</a>;
```

Comment:
Gosh that‘s truely never meant to work. This was a urgent GDPR fix by the way.

Reply:
babel is a wonderful thing 😄